pkgs/development/python-modules/quart/default.nix

```
@@ -0,0 +1,57 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitLab
+, click
+, aiofiles
+, werkzeug
+, jinja2
+, hypercorn
+, itsdangerous
+, blinker
+, mock
+, hypothesis
+, pytest
+, pytest-asyncio
+, pytest-cov
+, pytest-sugar
+}:
+
+buildPythonPackage rec {
+  pname = "quart";
+  version = "0.15.1";
+
+  src = fetchFromGitLab {
+    owner = "pgjones";
+    repo = pname;
+    rev = version;
+    sha256 = "036ip3lwpf4wkssmfpsdxnw46ncpafzwfswvsvkngbm9gnf1v4r7";
+  };
+
+  propagatedBuildInputs = [
+    click
+    aiofiles
+    jinja2
+    hypercorn
+    itsdangerous
+    blinker
+    werkzeug
+  ];
+
+  checkInputs = [
+    mock
+    hypothesis
+    pytest
+    pytest-asyncio
+    pytest-cov
+    pytest-sugar
+  ];
+
+  pythonImportsCheck = [ "quart" ];
+
+  meta = with lib; {
+    homepage = "https://pgjones.gitlab.io/quart/";
+    description = "An asyncio reimplementation of the popular Flask microframework API";
```

I would suggest to not mention `flask`. Perhaps something like:

```suggestion
    description = "Python ASGI web microframework";
```